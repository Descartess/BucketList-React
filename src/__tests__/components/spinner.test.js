import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '../../components/spinner';

describe(' Spinner renders correctly', () => {
  it('renders spinner correctly', () => {
    const rendered = renderer.create(<Spinner />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
