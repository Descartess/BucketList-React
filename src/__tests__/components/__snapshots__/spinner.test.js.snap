// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Spinner renders correctly renders spinner correctly 1`] = `
<div
  className="loader_display"
>
  <div
    className="loader"
  />
</div>
`;
